libs/perun/services/src/lib/dynamic-paginating.service.ts

```diff
@@ -102,6 +102,7 @@ export class DynamicPaginatingService {
         offset: pageSize * pageNumber,
         pageSize: pageSize,
         order: order,
+        selectedEvents: [],
```

Comment:
Please rebase it. After rebase I think this should use one of parameters (selectedEvents: string[]):
```suggestion
        selectedEvents: selectedEvents,
```